fe/fe-core/src/main/java/com/starrocks/service/FrontendServiceImpl.java

```diff
@@ -519,7 +519,7 @@ public TGetDBPrivsResult getDBPrivs(TGetDBPrivsParams params) throws TException
             PrivBitSet savedPrivs = entry.getPrivSet();
             String clusterPrefix = SystemInfoService.DEFAULT_CLUSTER + ClusterNamespace.CLUSTER_DELIMITER;
```

Comment:
I think we can remove the DEFAULT_CLUSTER also. The feature is useless but only has compatibility.

Reply:
thanks for review，up to now，we just remove default_cluster from default_cluster:db, and will remove default_cluster from default_cluster:role/user next step，so when default_cluster is used for role/user it is useful now